packages/playground/src/portal/twin_view.vue

```diff
@@ -52,16 +63,47 @@
 import { generatePublicKey } from "@threefold/rmb_direct_client";
 import { onMounted, ref } from "vue";
 
+import router from "../router";
 import { useProfileManager } from "../stores";
+import type { Farm } from "../types";
+import { getFarms } from "../utils/get_farms";
 import { getGrid } from "../utils/grid";
-
 const profileManager = useProfileManager();
 
 const editingTwin = ref(false);
 const relay = ref(profileManager.profile?.relay || "");
 const updateRelay = ref(false);
 const errorMsg = ref("");
+const openVotePopup = ref(false);
+const numberOfActiveProposals = ref(0);
+const userFarms = ref<Farm[]>();
+onMounted(async () => {
+  const profile = profileManager.profile!;
+  const grid = await getGrid(profile);
+  if (grid) {
+    userFarms.value = await getFarms(grid, { ownedBy: profile.twinId }, {});
+    if (userFarms.value.length > 0) {
```

Comment:
instead of having nested if, we can invert the checks and return. it will be more readable.